### DISCUSSION TITLE
只有一半的快捷键有用

### DISCUSSION THREAD

**Discussion by Linya-IronMan** [nshen/learn-neovim-lua#107] [Q&A] [answered]
<img width="487" alt="image" src="https://user-images.githubusercontent.com/77786952/188315149-b875c5ef-4fe7-4bf3-be88-1958346cea79.png">
比如这个左右比例控制 只有 C-] 是正常工作的，  C-[ 就无法工作。
但是我输入 :vertical resize -2<CR> 的时候，是可以的。

奇怪....

**Comment by nshen** [comment, marked as answer]
被更高级别的占用了吧，比如终端自带的快捷键
